### PR TITLE
Refactor/fr locale

### PR DESCRIPTION
**Pull request by RickardssonJ**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "@types/styled-components": "^5.1.26",
     "eslint": "8.27.0",
     "eslint-config-next": "13.0.3",
-    "prisma": "^4.10.1",
+    "prisma": "^4.11.0",
```
Comment (tasola) [inline package.json:46]: Probably no problem, but any reason why this is upgraded?

**Review by tasola — CHANGES_REQUESTED** [review]
Looks good! But I think some things are missing:

1. The French pages must be built in each [vuid].tsx's `getStaticPaths`, like the English and Spanish.
2. RecentUpdates (the logic in recent.ts) might have the be reworked. At least make sure to test it thoroughly (confirm that the recent updates are rendered in French if they have a French locale, else fallback to English). When I glance through the code I think it might work as is, but I'm not totally sure.
3. Test thoroughly that the filter results are working as intended as well. I'm more confident that this works than 2. though.

File: package.json (hunk 1 above)
Comment (RickardssonJ) [reply package.json:46]: No there's no reason. It just gave med the latest version when i installed it

**Review by tasola — APPROVED** [review]
A bit corrupted approval now, but I think it looks good 😇

**Pull request merged**